app/src/test/java/gov/va/vro/ClaimRepositoryTest.java

```diff
@@ -20,11 +23,20 @@ void insertQuery() {
     assertNotNull(veteran.getUpdatedAt());
 
     ContentionEntity contention1 = new ContentionEntity("c1");
-    contention1.addAssessmentResult(2);
+    AssessmentResultEntity ar = new AssessmentResultEntity();
+    Map<String, String> evidence = new HashMap<>();
+    evidence.put("medicationsCount", "10");
+    ar.setEvidenceCountSummary(evidence);
+    contention1.addAssessmentResult(ar);
     contention1.addEvidenceSummaryDocument("doc1", 1);
     contention1.addEvidenceSummaryDocument("doc2", 2);
     ContentionEntity contention2 = new ContentionEntity("c2");
-    contention2.addAssessmentResult(1);
+    Map<String, String> evidence2 = new HashMap<>();
+    AssessmentResultEntity ar2 = new AssessmentResultEntity();
+    evidence2.put("medicationsCount", "10");
+    ar2.setEvidenceCountSummary(evidence2);
+    ar2.setEvidenceCount(1);
+    contention2.addAssessmentResult(ar2);
     contention2.addAssessmentResult(2);
     var claim = TestDataSupplier.createClaim("123", "type", veteran);
     claim.addContention(contention1);
```

Comment:
We need code to retrieve the assessment result and verify that it contains the correct JSON